src/recaptcha-wrapper.js

```diff
@@ -35,6 +35,28 @@ export function createRecaptcha () {
       this.assertLoaded()
       this.wait().then(() => window.grecaptcha.execute(widgetId))
     },
+	
+	loadRecaptcha () {
+		let recaptchaDetected = false;
+		let recaptchaSrc = 'https://www.google.com/recaptcha/api.js?onload=vueRecaptchaApiLoaded&render=explicit';
+		let documentScripts = document.getElementsByTagName('script');
```

Comment:
```ts
const recaptchaDetected = document.querySelectorAll(`[src="${recaptchaSrc}"]`).length >= 1
```

Reply:
@texdc Nice suggestion - thanks.

@DanSnow if you can confirm your views on the switch Vs. vue-recaptcha property to override language, I'll update this PR this week.